integrations/tensorflow-keras/neptune_tensorflow_keras/__init__.py

```diff
@@ -82,8 +75,9 @@ class NeptuneCallback(Callback):
         You need to have Keras or Tensorflow 2 installed on your computer to use this module.
     """
 
-    def __init__(self, experiment: Run, base_namespace: Optional[str] = None):
+    def __init__(self, experiment: 'Run', base_namespace: Optional[str] = None):
         super().__init__()
+        from neptune.new import Run
```

Comment:
Why import here?

Reply:
We haven't release alpha client yet, so according to requirements, this package does not exist and it failds to build when imported

Reply:
It's going to be released today.

Reply:
Or tomorrow.